Directory.Build.props

```diff
@@ -104,6 +104,15 @@
     <IsTestProject Condition="$(MSBuildProjectName.EndsWith('Tests')) OR $(MSBuildProjectName.EndsWith('.Test'))">true</IsTestProject>
     <IsTestAssetProject Condition="$(RepoRelativeProjectDir.Contains('testassets'))">true</IsTestAssetProject>
     <IsSampleProject Condition="$(RepoRelativeProjectDir.Contains('sample'))">true</IsSampleProject>
+
+    <IncludeSource>false</IncludeSource>
```

Comment:
@natemcmaster wanted to confirm what to do with some of these properties.

Reply:
With the exception of DisableDeterministicSourceRoot , I think these are already defined in this file, so this is mostly adding duplicates.